desktop-style.css

```diff
@@ -0,0 +1,243 @@
+/* desktop version */
+@media screen and (min-width: 900px) {
```

Comment:
- [ ] [REQUIRED] As in the project requirements, please use media queries in your CSS file of any value in the range of 992px - 1100px. (990px is small)😊